### DISCUSSION TITLE
Is This Patched

### DISCUSSION THREAD

**Discussion by regularmovie** [3kh0/ext-remover#1002] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm trying to downgrade and I'm in 122 and I'm trying to go to 119 
and I saw this workaround from @bloohacked:

Press escape refresh and power
press tab
find where it says something like kernel version
the last number is your kernel version
if it is 0 or 1 you can downgrade to any version
if it is 2 you can downgrade between 112 and 119
if it is 3 you can downgrade between 120 and latest
source a recovery image for the version that you want and follow normal downgrading steps

I wanted to see if this would still work in 122

**Comment by nathblade16** [comment, marked as answer]
Yes, it would work in 122 since it is imposible to patch this, as long as your kernel version is 0, 1 or 2 you can go to 119

**Reply by v4pl** [reply]
Ya this should work depending on your kerner

**Comment by Possumxkings** [comment]
could you likes to messaging me or like take me out of this group chat
please?

**Reply by nathblade16** [reply]
there is a discord
